eitprocessing/variants/__init__.py

```diff
@@ -34,43 +30,20 @@ class Variant(ABC):
     description: str
     params: dict = field(default_factory=dict)
 
-    @staticmethod
-    def check_equivalence(a: T, b: T, raise_=False) -> bool:
-        """Check the equivalence of two variants
-
-        For two variants to be equivalent, they need to have the same class,
-        the same label, the same description and the same parameters. Only the
-        actual data can differ between variants.
-
-        Args:
-        - a (Variant)
-        - b (Variant)
-
-        Raises:
-        - NotEquivalent (only if raise_ is `True`) when a and b are not
-          equivalent on one of the attributes
-        """
-        cm = contextlib.nullcontext() if raise_ else contextlib.suppress(NotEquivalent)
-        with cm:
-            if not isinstance(a, b.__class__):
-                raise NotEquivalent(
-                    f"Variant classes don't match: {a.__class__}, {b.__class__}"
-                )
-
-            if (a_ := a.label) != (b_ := b.label):
-                raise NotEquivalent(f"EITDataVariant names don't match: {a_}, {b_}")
-
-            if (a_ := a.description) != (b_ := b.description):
-                raise NotEquivalent(
-                    f"EITDataVariant descriptions don't match: {a_}, {b_}"
-                )
-
-            if (a_ := a.params) != (b_ := b.params):
-                raise NotEquivalent(f"EITDataVariant params don't match: {a_}, {b_}")
-
-            return True
-
-        return False
+    def isequivalent(
+        self,
+        other: Self,
+        raise_=True,
+    ) -> bool:
+        EDV = "EITDataVariant"
```

Comment:
I guess because the arguments that are checked are all part of the `Variant` class (and I don't think that `EITDataVariant` adds any arguments that need to be checked), I think it should be here. I updated the string to reflect that it's any `Variant`.